File: chatwoot/csharp/src/OSEG.ChatwootExamples/CreateAContactExample.cs
using System;
using System.Collections.Generic;
using System.IO;
using System.Text.Json;

using Com.Chatwoot.Api;
using Com.Chatwoot.Client;
using Com.Chatwoot.Model;

namespace OSEG.ChatwootExamples;

public class CreateAContactExample
{
    public static void Run()
    {
        var config = new Configuration();

        var publicContactCreateUpdatePayload = new PublicContactCreateUpdatePayload(
        );

        try
        {
            var response = new ContactsAPIApi(config).CreateAContact(
                inboxIdentifier: "inbox_identifier_string",
                data: publicContactCreateUpdatePayload
            );

            Console.WriteLine(response);
        }
        catch (ApiException e)
        {
            Console.WriteLine("Exception when calling ContactsAPIApi#CreateAContact: " + e.Message);
            Console.WriteLine("Status Code: " + e.ErrorCode);
            Console.WriteLine(e.StackTrace);
        }
    }
}

File: chatwoot/typescript-node/src/CreateAContactExample.ts
import * as fs from 'fs';
import api from "chatwoot_client"
import models from "chatwoot_client"

const apiCaller = new api.ContactsAPIApi();

const publicContactCreateUpdatePayload: models.PublicContactCreateUpdatePayload = {
};

apiCaller.createAContact(
  "inbox_identifier_string", // inboxIdentifier
  publicContactCreateUpdatePayload, // data
).then(response => {
  console.log(response.body);
}).catch(error => {
  console.log("Exception when calling ContactsAPIApi#createAContact:");
  console.log(error.body);
});
